### DISCUSSION TITLE
Support for Arduino Uno R4

### DISCUSSION THREAD

**Discussion by mihallis** [tinygo-org/tinygo#4264] [Q&A]
Hi, i'm thinking about new project with Arduino UNO R4 with tinygo. It is possible to use tinygo with this board? Main difference between UNO r3 and r4 is now UNO R4 is based on ARM Cortex CPU instead of AVR 8-bit.

Thanks for answers.

**Comment by deadprogram** [comment]
Hello @CMDMichalKoval the R4 is based on the Renesas family of processors.

Please see my draft PR here https://github.com/tinygo-org/tinygo/pull/4265 not even close to complete, or even correct. Hopefully this can help us get started with adding this.
